### PR TITLE
Added contact

### PR DESCRIPTION
**Pull request by bekjon-me**
- added contact section

### CODE REVIEW

**Review by Ismailco — COMMENTED** [review]
Hi @bekjon-me, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that should be created using peer-programming, kindly make sure all students in the group are committing to this PR before requesting a  review

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -226,5 +226,34 @@ <h3>Skills</h3>
         </ul>
       </div>
     </section>
+
+    <section class="contact">
```
[the remark this answers is not in the document]
Comment (bekjon-me) [reply index.html:230]: no, it is same, I have checked

File: index.html [hunk 2]
```diff
@@ -226,5 +226,31 @@ <h3>Skills</h3>
...
```
Comment (1GORDON) [inline index.html:230]: Great job @bekjon-me 🤝

- [ ] [Optional] You may try to follow the Figma design to create a replica of your own. Please add the dark line to the bottom of your design. Take a look at the Figma design critically, you will see a black line.
- Figma design
footer figma

- Your design
footer yours

**Review by 1GORDON — APPROVED** [review]
# Status: APPROVED 🟢 

Hi @bekjon-me ✋



Your project is complete!  There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! :tada: :clap: :clap: :100:

## Highlights!
- Followed Gitflow ✔️ 
- Good commit messages ✔️
- Linters are passing ✔️
- Professional Pull request ✔️
- Professional Readme ✔️

## Optional suggestions

##### _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

##### Cheers and Happy coding!:clap::clap::clap:
##### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**